2019-03-01 10:03:18,375 - public_methods.py[line:108] - INFO: 执行更新数据时报错，报错为:[SSL: UNKNOWN_PROTOCOL] unknown protocol (_ssl.c:833)
2019-03-01 10:03:18,375 - public_methods.py[line:108] - INFO: 执行更新数据时报错，报错为:[SSL: UNKNOWN_PROTOCOL] unknown protocol (_ssl.c:833)
2019-03-01 10:03:18,375 - public_methods.py[line:108] - INFO: 执行更新数据时报错，报错为:[SSL: UNKNOWN_PROTOCOL] unknown protocol (_ssl.c:833)
2019-03-01 10:03:18,375 - public_methods.py[line:108] - INFO: 执行更新数据时报错，报错为:[SSL: UNKNOWN_PROTOCOL] unknown protocol (_ssl.c:833)
